site/content/en/blog/releases/1.45.0.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "1.45.0 - Kubernetes 1.31 Support, OKE Cluster Setup Documentation, bug fixes and more!"
+linktitle: "1.45.0"
+date: "2024-11-19"
+---
+
+This is the 1.45.0 release of Agones.
+
+In this release, we’ve introduced Terraform scripts and updated documentation for setting up Agones on Oracle Kubernetes Engine (OKE). Additionally, we’ve improved support for new Kubernetes versions, enhanced the Fleet Autoscaler, and introduced a new dashboard feature for GameServer state durations. Additionally, we've updated the Rust SDK and added a CI check for versioning in examples.
+
+- **Updated Kubernetes Version Support (v1.29, v1.30, v1.31)**: Agones now officially supports Kubernetes versions 1.29, 1.30, and 1.31, ensuring compatibility with the latest Kubernetes releases for improved performance and stability.
+
+- **Terraform Scripts and Documentation for OKE Cluster Setup**: New documentation has been added, including Terraform scripts to help users set up an OKE (Oracle Kubernetes Engine) cluster and install Agones.
+
+- **Rust SDK Updates**: All Rust SDK dependencies have been updated to the latest versions. This update brings performance improvements, bug fixes, and ensures compatibility with the newest features in Rust.
+
+- **Fleet Autoscaler Enhancements: List-based Scaling from Zero Replicas**: The Fleet Autoscaler now supports scaling game server fleets from zero replicas when using a type List Fleet Autoscaler. This improves the flexibility of scaling fleets with GitOps based tools.
+
+- **New Dashboard Feature for GameServer State Duration**: A new dashboard feature has been added that provides insights into the duration of GameServer states. This will help operators monitor and debug the lifecycle of game servers more efficiently.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.45.0" >README</a> for details on features, installation and usage.
```

Comment:
```suggestion
Check the <a href="https://github.com/googleforgames/agones/tree/release-1.45.0" data-proofer-ignore>README</a> for details on features, installation and usage.
```

Reply:
in release-1.45.0, there is no data-proofer-ignore right? 

this command is not working. "make del-data-proofer-ignore FILENAME=1.45.0-1.md" something we need to remove the word "-1" @igooch